lib/Doctrine/Migrations/Generator/SqlGenerator.php

```diff
@@ -30,10 +31,14 @@ class SqlGenerator
     /** @var AbstractPlatform */
     private $platform;
 
+    /** @var SqlFormatter */
+    private $sqlFormatter;
+
     public function __construct(Configuration $configuration, AbstractPlatform $platform)
     {
         $this->configuration = $configuration;
         $this->platform      = $platform;
+        $this->sqlFormatter  = new SqlFormatter(new NullHighlighter());
```

Comment:
I hesitated to have a ctor argument for this, but then that would mean do some mocking in the test, and "Don't mock what you don't own" :tm: 
When I see this, I think it could be cool to have a named constructor like `SqlFormatter::withoutHighlight()`